codegen/templates/http_client.tmpl

```diff
@@ -98,22 +99,79 @@ func (c *{{$clientName}}) {{title .Name}}(
 		respHeaders[k] = res.Header.Get(k)
 	}
 
-	res.CheckOKResponse({{.OKStatusCode.Code}})
+	res.CheckOKResponse([]int{
```

Comment:
Is this still necessary? It's covered by the default case in the switch check below